### ISSUE TITLE
Website: mylondon.news

### ISSUE THREAD

**Issue by DrKain** [DrKain/tidy-url#37]
**Website you would like added. Include multiple URLs if needed**

```
https://www.mylondon.news/whats-on/whats-on-news/cherry-blossom-tattoo-unpaid-apprenticeship-25822862?int_source=amp_continue_reading&int_medium=amp&int_campaign=continue_reading_button#amp-readmore-target
https://www.mylondon.news/whats-on/whats-on-news/tube-moment-of-love-photo-25810243?int_source=nba
```

**Bad parameters. Leave blank if you don't know.**

- int_source
- int_medium
- int_campaign

**Comment by elonyavist** [comment]
Hi @DrKain, I'd like to work on it

**Comment by elonyavist** [comment]
Looking the other PRs, I think I can create a branch directly from the main without forking the project

**Comment by DrKain** [comment]
Sure thing. You're welcome to work on any of the open issues.  
Just an extra note, your last 2 [commits](https://github.com/DrKain/tidy-url/commit/471489ea5982af90a44575934495492c6c236a9a) were not linked to your github account. It might be worth updating if you'd like your account to be featured on the [contributors](https://github.com/DrKain/tidy-url/graphs/contributors) page

**Comment by elonyavist** [comment]
Ohh thanks, I think I have some issue with my git config.

**Comment by DrKain** [comment]
Fixed in #43
